test/formatters/web-mode/prettier-babel.html.formatted

```diff
@@ -0,0 +1,112 @@
+                           Find and print the path to a configuration file for the given input file.
+  --ignore-path <path>     Path to a file with patterns describing files to ignore.
+                           Defaults to .prettierignore.
+  --plugin <path>          Add a plugin. Multiple plugins can be passed as separate `--plugin`s.
+                           Defaults to [].
+  --plugin-search-dir <path>
+                           Custom directory that contains prettier plugins in node_modules subdirectory.
+                           Overrides default behavior when plugins are searched relatively to the location of Prettier.
+                           Multiple values are accepted.
+                           Defaults to [].
+  --with-node-modules      Process files inside 'node_modules' directory.
+
+Editor options:
+
+  --cursor-offset <int>    Print (to stderr) where a cursor at the given position would move to after formatting.
+                           This option cannot be used with --range-start and --range-end.
+                           Defaults to -1.
+  --range-end <int>        Format code ending at a given character offset (exclusive).
+                           The range will extend forwards to the end of the selected statement.
+                           This option cannot be used with --cursor-offset.
+                           Defaults to Infinity.
+  --range-start <int>      Format code starting at a given character offset.
+                           The range will extend backwards to the start of the first line containing the selected statement.
+                           This option cannot be used with --cursor-offset.
+                           Defaults to 0.
+
+Other options:
+
+  --no-color               Do not colorize error messages.
+  --file-info <path>       Extract the following info (as JSON) for a given file path. Reported fields:
+                           * ignored (boolean) - true if file path is filtered by --ignore-path
+                           * inferredParser (string | null) - name of parser inferred from file path
+  -h, --help <flag>        Show CLI usage, or details about the given flag.
+                           Example: --help write
+  -u, --ignore-unknown     Ignore unknown files.
+  --insert-pragma          Insert @format pragma into file's first docblock comment.
+                           Defaults to false.
+  --loglevel <silent|error|warn|log|debug>
+                           What level of logs to report.
+                           Defaults to log.
+  --require-pragma         Require either '@prettier' or '@format' to be present in the file's first docblock comment
+                           in order for it to be formatted.
+                           Defaults to false.
+  --stdin-filepath <path>  Path to the file to pretend that stdin comes from.
+  --support-info           Print support information as JSON.
+  -v, --version            Print Prettier version.
+
+
```

Comment:
Hmmm, this seems not quite right! :P

Reply:
And that's why "Ensure all tests contain the proper text." is on the checklist :P